1436790/1438462: Part 50: Final 8 files with MSDN link changes.

DennisLee-DennisLee opened the pull request:
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

These are changes to instances I missed or needed to reconsider. There are a few more MSDN links, but most of them are to MSDN magazine articles that aren't on docs.microsoft.com.

DennisLee-DennisLee committed:
Final 9 files of MSDN link changes.

rpetrusha commented on docs/framework/wcf/feature-details/serializing-in-json-with-message-level-programming.md:
```diff
@@ -7,7 +7,7 @@ ms.assetid: 5f940ba2-57ee-4c49-a779-957c5e7e71fa
   
 ## Typed Message Programming  
- The <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> is used when the <xref:System.ServiceModel.Web.WebGetAttribute> or the <xref:System.ServiceModel.Web.WebInvokeAttribute> is applied to a service operation. Both of these attributes allow you to specify the `RequestFormat` and `ResponseFormat`. To use JSON for requests and responses set both of these to `WebMessageFormat.Json ...
+ The <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> is used when the <xref:System.ServiceModel.Web.WebGetAttribute> or the <xref:System.ServiceModel.Web.WebInvokeAttribute> is applied to a service operation. Both of these attributes allow you to specify the `RequestFormat` and `ResponseFormat`. To use JSON for requests and responses set both of these to `WebMessageFormat.Json ...
```
```suggestion
 The <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> is used when the <xref:System.ServiceModel.Web.WebGetAttribute> or the <xref:System.ServiceModel.Web.WebInvokeAttribute> is applied to a service operation. Both of these attributes allow you to specify the `RequestFormat` and `ResponseFormat`. To use JSON for requests and responses. set both of these to `WebMessageFormat.Json`.  In order to use JSON, you must use the <xref:System.ServiceModel.WebHttpBinding>, which automatically configures the <xref:System.ServiceModel.Description.WebHttpBehavior>. For more information about WCF serialization, see [Serialization and Deserialization](../../../../docs/framework/wcf/feature-details/serialization-and-deserialization.md). For more information about JSON and WCF, see [Service Station - An Introduction To RESTful Services With WCF](https://msdn.microsoft.com/magazine/dd315413.aspx).  
```
The second link appears to be broken.

rpetrusha commented on docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md:
```diff
@@ -185,7 +185,7 @@ You can see in the image above some example values based on the ViewModel types
   [*https://github.com/StackExchange/dapper-dot-net*](https://github.com/StackExchange/dapper-dot-net)
```
```suggestion
 <https://github.com/StackExchange/dapper-dot-net>
```

rpetrusha commented on docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md:
```diff
@@ -185,7 +185,7 @@ You can see in the image above some example values based on the ViewModel types
+  <https://msdn.microsoft.com/magazine/mt703432.aspx>
 
 - **ASP.NET Core Web API Help Pages using Swagger** \
   [*https://docs.microsoft.com/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio*](https://docs.microsoft.com/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio)
```
```suggestion
  <https://docs.microsoft.com/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio>
```

rpetrusha commented on docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/net-core-microservice-domain-model.md:
```diff
@@ -165,8 +165,8 @@ For example, in the preceding OrderAggregate code example, there are several pri
+  <https://msdn.microsoft.com/magazine/dn342868.aspx>
 
 - **Udi Dahan. How to create fully encapsulated Domain Models** \
   [*http://udidahan.com/2008/02/29/how-to-create-fully-encapsulated-domain-models/*](http://udidahan.com/2008/02/29/how-to-create-fully-encapsulated-domain-models/)
```
```suggestion
  <http://udidahan.com/2008/02/29/how-to-create-fully-encapsulated-domain-models/>
```

rpetrusha reviewed (changes requested):
Thanks for all of the work you've put into making these link changes, @DennisLee-DennisLee. I've left some suggestions, almost all unrelated to your PR.

rpetrusha commented:
//cc @mvelosop for visibility

Commit:
Apply suggestions from code review

Adding Ron's suggestions. Still need to revert my change to docs/framework/migration-guide/net-framework-4-migration-issues.md, as Maira had rejected a change to that link in PR #10421.

Co-Authored-By: DennisLee-DennisLee <v-dele@microsoft.com>

DennisLee-DennisLee committed:
Restoring old link for future migration per Maira in PR #10421.

mairaw commented on docs/framework/performance/index.md:
```diff
@@ -43,7 +43,7 @@ If you want to create apps with great performance, you should design and plan fo
 |Windows Presentation Foundation (WPF)|[WPF Performance Suite](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/aa969767(v=vs.100))|  
 |ASP.NET|[ASP.NET Performance Overview](https://docs.microsoft.com/previous-versions/aspnet/cc668225(v=vs.100))|  
-|Windows Forms|[Practical Tips for Boosting the Performance of Windows Forms Apps](https://msdn.microsoft.com/magazine/cc163630.aspx)|  
+|Windows Forms|[Winning Forms: Practical Tips for Boosting the Performance of Windows Forms Apps](https://web.archive.org/web/20060411224813/https://msdn.microsoft.com/msdnmag/issues/06/03/WindowsFormsPerformance/)|  
```
I'd just remove this 

```suggestion
```

mairaw commented on docs/framework/winforms/windows-forms-overview.md:
```diff
@@ -61,7 +61,7 @@ For step-by-step information about how to use these features, see the following
 |Using the <xref:System.Windows.Forms.BindingSource> component|[How to: Bind Windows Forms Controls with the BindingSource Component Using the Designer](../../../docs/framework/winforms/controls/bind-wf-controls-with-the-bindingsource.md)|
 |Working with [!INCLUDE[vstecado](../../../includes/vstecado-md.md)] data sources|[How to: Sort and Filter ADO.NET Data with the Windows Forms BindingSource Component](../../../docs/framework/winforms/controls/sort-and-filter-ado-net-data-with-wf-bindingsource-component.md)|
-|Using the Data Sources window|[Walkthrough: Displaying Data on a Windows Form](https://msdn.microsoft.com/library/f6e08c2c-c792-43de-adf3-3e52c0100225)|
+|Using the Data Sources window|[Walkthrough: Displaying Data on a Windows Form](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/wwh8ka92(v=vs.120))|
```
Let's change this to go to https://docs.microsoft.com/visualstudio/data-tools/bind-windows-forms-controls-to-data-in-visual-studio instead

mairaw reviewed (changes requested):
Looks good @DennisLee-DennisLee. Left a few comments to be addressed.

DennisLee-DennisLee committed:
Implementing Maira's suggestions.

mairaw reviewed (approved):
Thanks @DennisLee-DennisLee! Last one - woo hoo!

rpetrusha commented:
Thank you for making the additional changes, @DennisLee-DennisLee. I'll merge your PR now.

mvelosop commented:
Thank you @rpetrusha!
